src/dune_engine/build_system.ml

```diff
@@ -470,22 +470,10 @@ end = struct
       let* () = Targets.maybe_async (fun () -> Path.mkdir_p (Path.build dir)) in
       let is_action_dynamic = Action.is_dynamic action.action in
       let sandbox_mode =
-        match Action.is_useful_to_sandbox action.action with
```

Comment:
The function `Action.is_useful_sandbox` can be deleted. Or should we keep it?

Reply:
Let's delete it.